### PR TITLE
I'm trying add spanish filter

### PR DESCRIPTION
**Pull request by peter9811**
I'm trying to add the most common "official video" in Spanish
I need a guide to be able to do the rest

### CODE REVIEW

**Comment by inverse** [comment]
Hi @peter9811 thanks for taking the time to contribute! 

What are you trying to achieve? To apply your filter to an existing function e.g. `youtube` or introduce a new function that cleans up "official video" like strings? similar to how `removeRemastered` works?

**Comment by alexesprit** [comment]
You should add new rules to the `YOUTUBE_TRACK_FILTER_RULES` array to extend `youtube` filter function behavior.

**Comment by peter9811** [comment]
@alexesprit I think i did it how the correct form
Can you chechk it?

I don't have a lot of experience, but I want learn <3

**Comment by peter9811** [comment]
@inverse Yes, I try do this, but spanish... 

Maybe it's possible do a simple "examples" to copy and edit to add new rules?

**Comment by alexesprit** [comment]
> @alexesprit I think i did it how the correct form
> Can you chechk it?

It's correct now, but:

1. Your branch is outdated. You should rebase your branch onto the latest master, and apply your changes then.
2. You should add test cases for these rules (see [test/function/youtube.spec.ts](https://github.com/web-scrobbler/metadata-filter/blob/master/test/function/youtube.spec.ts) for details).

**Comment by peter9811** [comment]
> It's correct now, but:
> 
> 1. Your branch is outdated. You should rebase your branch onto the latest master, and apply your changes then.
> 2. You should add test cases for these rules (see [test/function/youtube.spec.ts](https://github.com/web-scrobbler/metadata-filter/blob/master/test/function/youtube.spec.ts) for details).

1. Can you explain me how rebase the band? I'm Windows user. I'm so noob on git, sorry
2. I'm going to check the test and try to add the filter to Spanish and may add something filter to english

**Comment by alexesprit** [comment]
> Can you explain me how rebase the band? I'm Windows user. I'm so noob on git, sorry

In this particular case it could be difficult for you to rebase. But if you're interesting in git, you can learn this command by reading [this document](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase).

I think it would be fine if you just add tests. I will rebase and merge the branch by myself.

**Comment by peter9811** [comment]
@alexesprit I recently add new filter... Maybe later I will able to add more

File: src/rules.js [hunk 1]
```diff
@@ -65,6 +65,18 @@ export const YOUTUBE_TRACK_FILTER_RULES = [
 	{ source: /^[/,:;~-\s"]+/, target: '' },
 	// trim trailing white chars and dash
 	{ source: /[/,:;~-\s"]+$/, target: '' },
+	
+	// Spanish
+	// (Videoclip Oficial) or [Videoclip Oficial]
+	{ source: /\s[([]Videoclip Of+icial[)\]]/i, target: '' },
+	// (Video Oficial) or [Video Oficial]
+	{ source: /\s[([]Video Of+icial[)\]]/i, target: '' },
```
Comment (alexesprit) [inline src/rules.js:73]: These two rules are redundant.

**Pull request closed without merging**